pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -76,9 +76,13 @@ func NewCmdAddRoleToUser(name, fullName string, f *clientcmd.Factory, out io.Wri
 	saNames := []string{}
 
 	cmd := &cobra.Command{
-		Use:     name + " ROLE (USER | -z SERVICEACCOUNT) [USER ...]",
-		Short:   "Add a role to users or serviceaccounts for the current project",
-		Long:    `Add a role to users or serviceaccounts for the current project`,
+		Use:   name + " ROLE (USER | -z SERVICEACCOUNT) [USER ...]",
+		Short: "Add a role to users or serviceaccounts for the current project",
+		Long: fmt.Sprintf(`Add a role to users or serviceaccounts for the current project.
```

Comment:
This command needs a better long description anyway - can you add a paragraph explaining what this method does, how you find roles, and how it can fail?